samtranslator/model/apigatewayv2.py

```diff
@@ -344,7 +346,9 @@ def _get_identity_source(self, auth_identity: Dict[str, Any]) -> List[str]:
         return identity_source
 
     @staticmethod
-    def _get_jwt_configuration(props: Optional[Dict[str, Union[str, List[str]]]]) -> Optional[JwtConfiguration]:
+    def _get_jwt_configuration(
+        props: Optional[Dict[str, Union[str, List[str]]]], logical_id: str
```

Comment:
nit: missing the comment for the new parameter

Reply:
thnx, added comment